go.mod

```diff
@@ -9,7 +9,7 @@ require (
 	github.com/fortytw2/leaktest v1.3.0
 	github.com/fsnotify/fsnotify v1.4.9
 	github.com/gocarina/gocsv v0.0.0-20190927101021-3ecffd272576
-	github.com/google/go-cmp v0.5.5
+	github.com/google/go-cmp v0.5.8
 	github.com/haproxytech/config-parser/v4 v4.0.0-rc1
 	github.com/openshift/api v0.0.0-20211201130627-34f305c3af47
```

Comment:
Does this need a bump as well? It looks like it was done for cluster-ingress-operator in [#829](https://github.com/openshift/cluster-ingress-operator/pull/829)

Reply:
Sorry just saw this comment. Hmmm, that's strange. I didn't explicitly do the bump of openshift/api in cluster-ingress-operator and nor did I do it here. Not sure why this didn't bump, but maybe it's a dependency chain that openshift/router is not triggering?
Do you think I should bump openshift/api to it's K8S API bump regardless? I'm not sure.

Reply:
I'd be inclined to say "no" only because if the router doesn't need the bump, then we probably don't gain anything by bumping it aside from keeping the future delta of the API changes smaller. WDYT @Miciah ?

Reply:
Bumping openshift/api isn't necessary.  There are no changes in it that would directly affect openshift/router.